css/style.css

```diff
@@ -7,26 +7,365 @@ body {
     font-size: 0.8rem;
     background-color: #fff9ff;
     margin: 0;
-    padding-left: 1rem;
+    padding: 0;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr auto 1fr;
```

Comment:
Maybe if you change the grid-template-rows to : `auto 1fr auto` it could fix the issue with the body getting too long